WalkingChallenge/Leaderboard.swift

```diff
@@ -120,7 +120,12 @@ extension LeaderBoard: UITableViewDataSource {
     else { return UITableViewCell() }
 
     cell.configure(entry)
-    return cell as! UITableViewCell
+    // TODO: Sami fix this to use actual CellInfo pattern.
+    if let cell = cell as? UITableViewCell {
+      return cell
+    } else {
+      return UITableViewCell()
+    }
   }
```

Comment:
Cant this be written as:

    return cell as? UITableViewCell ?? UITableViewCell()